include/odp/api/spec/reassembly.h

```diff
@@ -0,0 +1,120 @@
+/* Copyright (c) 2021, Marvell
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/**
+ * @file
+ *
+ * ODP REASSEMBLY API
+ */
+
+#ifndef ODP_API_SPEC_REASSEMBLY_H_
+#define ODP_API_SPEC_REASSEMBLY_H_
+#include <odp/visibility_begin.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @defgroup odp_reassembly ODP REASSEMBLY
+ *  Reassembly
+ *  @{
+ */
+
+/**
+ * Reassembly capabilities
+ *
+ */
+typedef struct odp_reass_capability_t {
+	/** Reassembly offload for both IPv4 and IPv6 packets */
+	odp_bool_t ip;
```

Comment:
@psavol @JannePeltonen Should the odp_bool_t here be replaced with odp_support_t?

Reply:
We are OK with bool. Support enum (additional preferred option) does not seem useful.